infra/gcp/terraform/k8s-infra-public-pii/main.tf

```diff
@@ -85,6 +85,25 @@ resource "google_storage_bucket" "audit-logs-gcs" {
   }
 }
 
+// service account for running ASN etl pipeline job
+resource "google_service_account" "asn_etl" {
+  project      = local.project_id
+  account_id   = "asn-etl"
+  display_name = "asn-etl"
+}
+
+// service account for running ASN etl pipeline job
+data "google_service_account" "public_pii_asn_etl" {
+  account_id = "asn-etl"
+  project    = local.project_id
+}
```

Comment:
Not needed. The resource `google_service_account.asn_etl` create the service account.

Reply:
This is used for accessing referring to the ServiceAccount in the RoleBindings below.